package-lock.json

```diff
@@ -0,0 +1,9075 @@
+{
```

Comment:
Remove this file from the diff (use yarn not npm)

Reply:
Removed file from git